packages/core-components/src/layout/Sidebar/Items.tsx

```diff
@@ -311,6 +311,7 @@ export function SidebarSearchField(props: SidebarSearchFieldProps) {
   const handleEnter: KeyboardEventHandler = ev => {
     if (ev.key === 'Enter') {
       search();
+      ev.preventDefault();
```

Comment:
Just a small nit here, can we call `ev.preventDefault` before we call search.

Also some tests would be great!

Reply:
Sure, I'll try it out. What about handleItemClick() a few rows below, https://github.com/backstage/backstage/blob/master/packages/core-components/src/layout/Sidebar/Items.tsx#L329 , should the order be changed there as well?

I might need some assistance writing tests for the search functionality. I'll see what I can do first :)

Reply:
> Just a small nit here, can we call `ev.preventDefault` before we call search.

Fixed!

> Also some tests would be great!

Can you give me some advice how to write a test for this? Any pointers would be appreciated. Thanks

Reply:
@benjdlambert, thanks for the pointers! 

A test has been added. Let me know if there's anything else I should do :+1: